card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
> I don't think we need a decko-rails command

We don't need it, but I think we want one.  It teaches the decko way.  And for decko experts, you don't really have to learn anything new.  I would personally have a hard time trying to learn the commands you're proposing.

> I'm trying to reduce the number of files to a minimum

I want to keep those down, too, but it's more important to have an awesome user interaction. You can start with generators if you want, but when we're ready to promote decko-rails, we'll want something much better than than `rails generate card` which is totally baffling.  Rails isn't generating anything; decko-rails is.  And what's being generated is not a card. (rails g migration creates a migration, rails g engine creates an engine.  controller, model, view.... this would be a weird generator that doesn't generate what it says it does)  

> I never wanted to duplicate things, but I also don't really want to reference wagn tasks from decko-rails. Can we move them to the decko now? (or will it be deck for rake tasks?) So decko:migrate and sub-tasks should be the same for both cases (aka wagn:migrate).

To me it's obvious that decko-rails should depend upon decko (currently wagn).   And decko/wagn should not depend upon decko-rails. I really don't get why this is in question.  We want people to find decko via decko-rails and not the other way around.   decko-rails a plugin for putting decko in your rails app.  It's just so clear that it's a separate gem.

As for how the commands work, the plan is:

The tasks will all subsitute wagn with decko:
`wagn new` = create app deck now
`decko new` = create app deck future

What I've been proposing is this:
`decko-rails new` = create engine deck soon

We could do it other ways, like
`wagn new` (soon `decko new`) is smart and recognizes when it's in a non-wagn rails app.  But this is the kind of logic I don't think belongs in the wagn gem.  

~~~
most other CLI options will translate directly to rake tasks
`wagn seed`  = `rake wagn:seed` = create and populate app deck database now/soon
`decko seed` = `rake decko:seed` = create and populate app deck database future

Again I'm proposing that we follow something like this pattern with:
`decko-rails seed`, but  I don't have any deep proposals for the namespacing of the rake tasks triggered.

> Can we move them to the decko now? (or will it be deck for rake tasks?) So decko:migrate and sub-tasks should be the same for both cases (aka wagn:migrate).

I'm definitely fine with having both namespaces handled in the decko-rails gem.

To me, it seems like the big next step is getting it so that wagn no longer depends on decko-rails.   

I keep saying this, but why don't you try hacking together an engine?  (don't worry about generators or special CLI -- just get it to work) It doesn't sound like we've even been playing with engines at all, and at this point that seems overdue, no?